server/services/interventionsService.test.ts

```diff
@@ -130,14 +129,116 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
           },
         },
       })
-    })
 
-    it('returns the updated referral', async () => {
       const referral = await interventionsService.patchDraftReferral('token', 'dfb64747-f658-40e0-a827-87b4b0bdcfed', {
         completionDeadline: '2021-04-01',
       })
       expect(referral.id).toBe('dfb64747-f658-40e0-a827-87b4b0bdcfed')
       expect(referral.completionDeadline).toBe('2021-04-01')
     })
+
+    it('returns the updated referral when selecting the complexity level', async () => {
+      await provider.addInteraction({
+        state: 'a draft referral with ID dfb64747-f658-40e0-a827-87b4b0bdcfed exists',
+        uponReceiving: 'a PATCH request to update the complexity level ID',
+        withRequest: {
+          method: 'PATCH',
+          path: '/draft-referral/dfb64747-f658-40e0-a827-87b4b0bdcfed',
+          headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+            Authorization: 'Bearer token',
+          },
+          body: { complexityLevelId: 'd0db50b0-4a50-4fc7-a006-9c97530e38b2' },
+        },
+        willRespondWith: {
+          status: 200,
+          body: {
+            id: Matchers.like('dfb64747-f658-40e0-a827-87b4b0bdcfed'),
+            complexityLevelId: 'd0db50b0-4a50-4fc7-a006-9c97530e38b2',
```

Comment:
Semantically, PATCHing a `serviceCategory.name` might raise the expectation to update the category's name.

Reply:
> In other words, do we consider a "low complexity" choice on a category different than another "low complexity" choice on another service category?

Maybe they can be modelled the same way, but from a presentation point of view they differ in that the `description` text (displayed as a hint beneath each radio button) is service category-specific.

Reply:
So we'd make a request for the service category instead, like `/service-categories/${serviceCategoryId}` that would return an object with a `name` field rather than store the`serviceCategory` on the `DraftReferral`? Would we want to store `ComplexityLevel`s on it too, and replace the call to `/service-categories/${serviceCategoryId}/complexity-levels` and just get them from the `serviceCategory` instead?) That could make sense - I wonder if anyone has any strong objections, as it'll be a bit of a re-work.